packages/cli/templates/init/Dockerfile.hbs

```diff
@@ -10,22 +10,23 @@
 ## description     : Dockerfile for TsED Application                         ##
 ## author          : TsED team                                               ##
 ## date            : 2021-04-14                                              ##
-## version         : 1.1                                                     ##
+## version         : 1.2                                                     ##
 ##                                                                           ##
 ###############################################################################
 ###############################################################################
 FROM node:14-alpine
 
-RUN apk update && apk add build-base git python
+RUN apk update && apk add build-base git python3
 
 COPY package.json .
 COPY yarn.lock .
+
+RUN yarn install --production
+
 COPY ./src ./src
 COPY ./dist ./dist{{#or swagger oidc}}
 COPY ./views ./views{{/or}}
 
-RUN yarn install --production
-
```

Comment:
Ensures that code change does not trigger dependency installation on each build.